### ISSUE TITLE
Initial TODO List

### ISSUE THREAD

**Issue by allenanie** [windweller/Trident#1]
To-be-filled

**Comment by allenanie** [comment]
1. Evaluate BLEU score and possibly other evaluation metrics on generative model
2. Read paragraph/document compression literature
3. Finish data preprocessing
4. Build Tensorflow model on 1 encoder 1 decoder
5. Build Tensorflow model on 4 encoders 1 decoder --- (save to last)
6. Decoder with inner-attention? (same reason for why it's helpful for SNLI)
7. Discard pre-trained decoder vs. keep pre-trained decoder
8. Add attention mechanism
9. Add Batch Normalization, and add recurrent dropout (normal dropout)
10. Start writing the paper?

**Comment by erindb** [comment]
also try out different GLOVE dimensions

**Comment by erindb** [comment]
also later, try out using PER and ORG tags instead of random vectors for words that aren't in the pretraining

**Comment by erindb** [comment]
note to self about other evaluations metrics: maybe stuff for paragraph/document compression?
